api/src/main/java/uk/gov/dhsc/htbhf/claimant/entity/Claim.java

```diff
@@ -66,4 +70,18 @@
     @ToString.Exclude
     @Valid
     private Claimant claimant;
+
+    public void updateClaimStatus(ClaimStatus newStatus) {
+        if (claimStatus != newStatus) {
+            claimStatusTimestamp = LocalDateTime.now();
+        }
+        claimStatus = newStatus;
+    }
+
+    public void updateEligibilityStatus(EligibilityStatus newStatus) {
```

Comment:
We're not actually using this method yet (i.e. we're never updating eligibilityStatus).